src/meta-assistant/timebank/timebank-api.ts

```diff
@@ -0,0 +1,116 @@
+import { WeeklyCombinedData } from "src/types/meta-assistant/index";
+import { Person, DailyEntriesApi, PersonsApi, DailyEntry, Timespan, PersonTotalTime } from "src/generated/client/api";
+
+/**
+ * Namespace for Timebank API
+ */
+namespace TimebankApi {
```

Comment:
Generated API would probably be better idea

Reply:
Agreed, this could be another quite nice trainee task if it's not urgent